src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { UniversalNavbarExpanded } from './UniversalNavbarExpanded'
+
+describe('<UniversalNavbarExpanded>', () => {
+  describe('matches snapshot', () => {
+    it('with no arguments', () => {
```

Comment:
Can we describe what we should be seeing visually with no arguments

Reply:
yeah this is leftover from v1, i'll write out the difference

Reply:
marked in TODOs for fastfollow